echopype/echodata/sensor_ep_version_mapping/v05x_to_v06x.py

```diff
@@ -985,7 +984,9 @@ def _rename_vendor_group(ed_obj):
     The function directly modifies the input EchoData object.
     """
 
-    ed_obj._tree["Vendor"].name = "Vendor_specific"
+    node = ed_obj._tree["Vendor"]
+    ed_obj._tree["Vendor"].orphan()
```

Comment:
@lsetiawan can you explain to me why this is necessary?

Reply:
This is needed to remove the old group... seems to work this way now. Maybe can look into how to improve it in future.